Podfile

```diff
@@ -17,6 +17,7 @@ pod 'Crashlytics', '3.14.0'
 pod 'Fabric', '1.10.2'
 pod 'Sparkle', '1.22.0', :configurations => ['Release']
 pod 'MIKMIDI', '1.7.0'
+pod 'SocketRocket'
```

Comment:
Can we pin this pod to a specific version please? I don't mind what the version number is - whatever the latest version that works for you, but it'll save us pain in the future if we have a pin.

Reply:
Done.